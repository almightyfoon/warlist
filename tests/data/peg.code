c2501b_-0O0J1S1U1G3q2e3k3E3C3t3z30iq3O3G3H___-0ObV221G1H3q3Ee41G3U3C3z3B303G3H
