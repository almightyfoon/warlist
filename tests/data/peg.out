https://conflictchamber.com/#c2501b_-0O0J1S1U1Gme2e3k3E3C3t3z30iq3O3G3H___-0ObV221G1Hme3Ee41G3U3C3z3B303G3H

Protectorate Army - 75 / 75 points
[Theme] The Faithful Masses

[High Reclaimer 1] The High Reclaimer [+32]
 - Castigator [12]
 - Crusader [10]
 - Redeemer [11]
 - Hierophant [3]
Avatar of Menoth [20]
Allegiant of the Order of the Fist [3]
High Paladin Dartan Vilmon [0(6)]
The Covenant of Menoth [4]
Vassal Mechanik [1]
Vassal of Menoth [3]
Choir of Menoth (max) [6]
Horgenhold Forge Guard (max) [16]
 - Attendant Priest [3]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]

Protectorate Army - 75 / 75 points
[Theme] The Faithful Masses

[Kreoss 3] Intercessor Kreoss [+28]
 - Reckoner [16]
 - Redeemer [11]
 - Revenger [10]
 - Hierophant [3]
High Paladin Dartan Vilmon [0(6)]
Initiate Tristan Durant [4]
 - Redeemer [11]
 - Sanctifier [14]
The Covenant of Menoth [4]
Vassal of Menoth [3]
Wrack [1]
Choir of Menoth (max) [6]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [5]
